test/README.md

```diff
@@ -1,26 +1,24 @@
 ## Single test runner
 
 ### How to add a new test file
+The single test runner supports automatic test discovery. We only need to put a test file into a `test` subfolder, next to the files we want to test.
+The runner uses this glob pattern `@(client|server|test)/**/test/*.@(js|jsx)` to find test files.
 
-Single test runner supports automatic test discovery. You only need to put your test file into `test` subfolder next to file under test.
-Runner uses this `glob` pattern `@(client|server|test)/**/test/*.@(js|jsx)` to find test files.
+We should use the same file names as the implementation files for the tests.
+Example: if we want to cover with unit tests `hello-world/index.jsx` file, we should name a test file `hello-world/test/index.jsx`.
 
-We should use the same name for test file like we named test we want to cover with tests.
+If we ever need to add non-test files to a `test` folder, we should put them in a deeper level. Common choices are:
 
-If we ever need to add non-test files to `test` folder we should put them on a deeper level. Common choices are:
 * `test/mocks/name.js` for test mocks
 * `test/fixtures/name.js` for test data
 
 ### How to run single test runner
 
-You can run all tests from the root folder using the `npm test` command.
+We have one command to run the tests `make test` from the root folder. It’s now an alias to the `npm test` command.
```

Comment:
It's okey to skip an alias part, and mention well known execution command. We can change that in the future when we decide to migrate fully to npm scripts :)